docs/UserGuide.md

```diff
@@ -956,6 +956,201 @@ Important notes:
 > > <img src="images/unicash/command-outputs/addTransactionFailedOutput3.png" width="1000" />
 
 #### 4.1.2 EditTransactionCommand
+Edits an existing transaction in UniCa$h.
+
+Command: `edit_transaction INDEX [n/NAME] [type/TYPE] [amt/AMOUNT] [dt/DATETIME] [l/LOCATION] [c/CATEGORY]`
+
+Command Options:
+
+| Option Name | Optional? | Purpose                                                                                       |
+|-------------|-----------|-----------------------------------------------------------------------------------------------|
+| INDEX       | No        | (One-based) index of the transaction to edit.                                                 |
+| n/          | Yes       | Name of the transaction.                                                                      |
+| type/       | Yes       | Transaction type of transaction.<br/>Valid types are `income` and `expense`.                  |
+| amt/        | Yes       | Monetary amount of transaction. Has to be a positive value.                                   |
+| dt/         | Yes       | Date and time where transaction was made.<br/>Defaults to current date time if not specified. |
+| l/          | Yes       | Location where transaction was made.<br/>Defaults to `''` if not specified.                   |
+| c/          | Yes       | Category tagged to that transaction.<br/>No categories tagged if not specified.               |
+
+Important notes:
```

Comment:
Maybe this would be better named as constraints? While it is good to be detailed, this section is a bit too long and could be quite overwhelming for users. I would suggest either moving most of these to an appendix, or have group sections for clarity.

Reply:
I've followed your suggestion in the other comment to move these constraints to a new "Remark" column!